blocks/form/form.js

```diff
@@ -394,6 +415,7 @@ export async function generateFormRendition(panel, container, getItems = (p) =>
 
   const children = await Promise.all(promises);
   container.append(...children.filter((_) => _ != null));
+  decoratePanelContainer(container, panel);
```

Comment:
Also, this container generateFormRendition Method seems very unituitive.
From what i can understand , it takes a containerDefinition as the first argument and the containerElement as the second argument.
the container can be either a panel or a form. But the method argument names are (panel,container). Can you update this as well.

Reply:
I agree with the poor naming. I've changed the signature of the function I've written but I'm reluctant to change it in generateFormRendition for now.